src/data.jsx

```diff
@@ -0,0 +1,33 @@
+const Items = [
```

Comment:
I would recommend that you use `camelCase` for common variables,  and only use `PascalCase` for components. for very important constants, use `CAPITALS`
while `Items` here is a constant, it represents data that will be gotten from a database in future for example. so its better practice to use camel case there